### ISSUE TITLE
[CI] PingAndInfoIT#testXPackInfo fails on 6.x

### ISSUE THREAD

**Issue by dnhatn** [elastic/elasticsearch#34386]
PingAndInfoIT#testXPackInfo fails on 6.x branch. I can reproduce this locally.

```
 ./gradlew :client:rest-high-level:integTestRunner \
  -Dtests.seed=10E186DE107D69A4 \
  -Dtests.class=org.elasticsearch.client.PingAndInfoIT \
  -Dtests.method="testXPackInfo" \
  -Dtests.security.manager=true \
  -Dtests.locale=he \
  -Dtests.timezone=Europe/Volgograd \
  -Dcompiler.java=11 \
  -Druntime.java=8
```

Failure:

```
FAILURE 0.67s | PingAndInfoIT.testXPackInfo <<< FAILURES!
   > Throwable #1: org.junit.ComparisonFailure: expected:<[]6.5.0> but was:<[based on ]6.5.0>
   >    at __randomizedtesting.SeedInfo.seed([10E186DE107D69A4:F8BD46DAC76111C3]:0)
   >    at org.elasticsearch.client.PingAndInfoIT.testXPackInfo(PingAndInfoIT.java:87)
   >    at java.lang.Thread.run(Thread.java:748)
```

CI instances:
- https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-unix-compatibility/os=opensuse/53/console
- https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-unix-compatibility/os=amazon/53/console
- https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-unix-compatibility/os=debian/53/console

**Comment by droberts195** [comment]
This test was muted on 10th October 2018, when we switched over from building ml-cpp snapshots on the old Prelert Jenkins to building on infra's Jenkins at elasticsearch-ci.elastic.co.  For a few days there was a problem where ml-cpp built within a Docker container on infra's setup had a crazy idea of the state of the git repo within the Docker container because it was relying on symlinks to directories outside of the Docker container to optimize the cloning process (inherited from the elasticsearch repo's worker setup).  We fixed this in elastic/ml-cpp#270 after debugging using elastic/ml-cpp#269.  The full story is in elastic/infra#7464.

It's unfortunate that nobody in the ML team was pinged when this test was muted, as we could have dealt with unmuting at the same time as we fixed the underlying problem.  (In fact, you can see from the dates of those other issues and PRs that I linked that we didn't notice there was a problem for 10 days after this ES test failure revealed it.)

**Comment by dakrone** [comment]
This has been re-enabled for master and 6.x, going to close this issue.
